#goal definition
cob_script_server/ScriptState state
---
#result definition
int16 return_value
---
#feedback definition
